contribs/gmf/src/datasource/fileGroup.js

```diff
@@ -0,0 +1,39 @@
+// The MIT License (MIT)
+//
+// Copyright (c) 2017-2020 Camptocamp SA
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of
+// this software and associated documentation files (the "Software"), to deal in
+// the Software without restriction, including without limitation the rights to
+// use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+// the Software, and to permit persons to whom the Software is furnished to do so,
+// subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+// FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+// COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+// IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+// CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import angular from 'angular';
+
+/**
+ * @type {angular.IModule}
+ * @hidden
+ */
+const module = angular.module('gmfDatasourceFileGroup', []);
+
+/**
+ * @typedef {Object} DatasourceFileGroup
+ * @property {import("ngeo/datasource/FileGroup.js").default|null} fileGroup
+ */
+
+module.value('gmfDatasourceFileGroup', {
```

Comment:
Also, can you describe in 1-2 lines what this value is for ? (I know, generally we (sadly) don't describe values but as this file have nothing else than this value, a short description could be helpful)
(And, or add a description in the DatasourceFileGroup typedef)

Reply:
That's an excellent idea.  I'll do this right away.